hail/hail-ci-build.sh

```diff
@@ -194,6 +194,22 @@ test_gcp() {
     touch ${GCP_SUCCESS}
 }
 
-test_project
-
-test_gcp > ${GCP_LOG}
+test_project &
+PID1=$!
+echo "test_project PID is ${PID1}"
```

Comment:
Technically you don't need the echo because you'll pick up the PID assignment:

```
$ cat foo.sh
#!/bin/bash
set -ex

echo oh hi &
PID=$!

echo $PID
$ bash foo.sh
+ PID=25464
+ echo 25464
25464
+ echo oh hi
oh hi
$ 
```

Reply:
ah right!